### PR TITLE
Onboard to QE Toolbelt Catalog

### PR DESCRIPTION
**Pull request by thrix**
See for more details about QE Toolbelt catalog:

https://console.one.redhat.com/docs/quality-community/Component/qe-toolbelt-catalog

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    # https://backstage.io/docs/features/software-catalog/descriptor-format#links-optional
+    links:
+        - title: user guide
+          url: https://docs.google.com/document/d/1kRBXCGVopImAJYX7KMZA0RJFhW2a_U03yYFJ_2Bg-eA/edit?usp=sharing
+          icon: techdocs
+        - title: code
+          url: https://github.com/RedhatQE/newa
+          icon: github
+        - title: pypi
+          url: https://pypi.org/project/sentaku/
```
Comment (kkaarreell) [inline catalog-info.yaml:19]: What's this?

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        - title: copr repository
+          url: https://copr.fedorainfracloud.org/coprs/packit/RedHatQE-newa-main/
+          icon: externalLink
+          type: copr
+
+    # https://console.one.redhat.com/docs/quality-community/Component/qe-toolbelt-catalog/onboarding/#tags
+    tags:
+        - jira
```
Comment (kkaarreell) [inline catalog-info.yaml:27]: Add errata tag as well?

File: catalog-info.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+metadata:
+    name: Newa
+    description: |
+      New Errata Workflow Automation.
+      Automation tool for automated testing of Red Hat Erratums using tmt, Testing Farm, ReportPortal and Jira.
+
+    # https://backstage.io/docs/features/software-catalog/descriptor-format#links-optional
+    links:
+        - title: user guide
+          url: https://docs.google.com/document/d/1kRBXCGVopImAJYX7KMZA0RJFhW2a_U03yYFJ_2Bg-eA/edit?usp=sharing
```
Comment (kkaarreell) [inline catalog-info.yaml:13]: Is it wise to share URL of an internal document? Should I expect access requests from public?

File: catalog-info.yaml (hunk 1 above)
Comment (thrix) [reply catalog-info.yaml:19]: a typo :)

File: catalog-info.yaml (hunk 3 above)
Comment (thrix) [reply catalog-info.yaml:13]: No, this file is for internal catalog, I would put it here :)

File: catalog-info.yaml (hunk 1 above)
Comment (thrix) [inline catalog-info.yaml:19]:
```sugestion
```

File: catalog-info.yaml (hunk 2 above)
Comment (thrix) [reply catalog-info.yaml:27]: Can you approve https://gitlab.cee.redhat.com/quality-innovators/qe-toolbelt/-/merge_requests/11 ?

File: catalog-info.yaml (hunk 1 above)
Comment (kkaarreell) [reply catalog-info.yaml:19]: But how Is this related to newa?

File: catalog-info.yaml (hunk 3 above)
Comment (kkaarreell) [reply catalog-info.yaml:13]: But the link would be in a public repo.

File: catalog-info.yaml (hunk 2 above)
Comment (thrix) [reply catalog-info.yaml:27]:
```suggestion
        - errata
        - jira
```

File: catalog-info.yaml (hunk 3 above)
Comment (thrix) [reply catalog-info.yaml:13]: yes, I do not see an issue with that, but a good discussion point :)

Comment (thrix) [reply catalog-info.yaml:13]: docs.google.com is no secret is it?

This file will be consumed only by internal RH users. We will replace once there is a nice user guide somewhere here and public?

Comment (thrix) [reply catalog-info.yaml:13]: Added to next meeting agenda :)

**Review by kkaarreell — APPROVED** [review]

**Pull request merged**